relay-quotas/src/rate_limit.rs

```diff
@@ -286,6 +286,21 @@ impl RateLimits {
     pub fn longest(&self) -> Option<&RateLimit> {
         self.iter().max_by_key(|limit| limit.retry_after)
     }
+
+    /// Returns the most relevant rate limit.
+    ///
+    /// The most relevant rate limit is the longest rate limit for events and if there is no
```

Comment:
You could clarify in the doc string that this is for error events only.

Reply:
yep